dev/com.ibm.ws.concurrency.policy/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -0,0 +1,45 @@
+concurrencyPolicy.desc=A set of behaviors and constraints to be applied for tasks that are capable of asynchronous execution. For example maximum concurrency and maximum queue size.
+
+# attributes
+
+expedite=Expedite
+expedite.desc=Specifies a core number of tasks to aim to run concurrently by expediting requests to the global thread pool. This provides no guarantee that this many tasks will run concurrently. If the expedite value is updated while in use, the update goes into effect gradually as previous expedited and non-expedited requests complete.
+
+max=Maximum concurrency
+max.desc=Specifies the maximum number of tasks that can be running at any given point in time. The default is unbounded, up to Integer.MAX_VALUE. Maximum concurrency can be updated while tasks are in progress. If the maximum concurrency is reduced below the number of concurrently executing tasks, the update goes into effect gradually as in-progress tasks complete rather than causing them to be canceled.
+
+maxPolicy=Maximum policy
+maxPolicy.desc=Indicates whether to loosely or strictly enforce maximum concurrency for tasks that run on the task submitter's thread. Tasks can run on the task submitter's thread when using the untimed invokeAll method, or, if only invoking a single task, the untimed invokeAny method. If the run-if-queue-full attribute is configured, it is also possible for tasks to run the task submitter's thread when using the execute and submit methods. In all of these cases, this attribute determines whether or not running on the submitter's thread counts against the maximum concurrency.
+maxPolicy.loose.desc=Maximum concurrency is loosely enforced, in that tasks are allowed to run on the task submitter's thread without counting against maximum concurrency.
+maxPolicy.strict.desc=Maximum concurrency is strictly enforced, in that tasks that run on the task submitter's thread count towards maximum concurrency. This policy does not allow running on the task submitter's thread when already at maximum concurrency.
+
+maxQueueSize=Maximum queue size
+maxQueueSize.desc=Specifies the maximum number of tasks that can be queued for execution at any given point in time. As tasks are started, canceled, or aborted, they are removed from the queue. When the queue is at capacity and another task is submitted, the behavior is determined by the maximum wait for enqueue and run-if-queue-full attributes. To ensure that a specific number of tasks can be queued within a short interval of time, use a maximum queue size that is at least as large as that amount. The default maximum queue size is unbounded, up to Integer.MAX_VALUE. Maximum queue size can be updated while tasks are in progress and/or queued for execution. If the maximum queue size is reduced below the current number of queued tasks, the update goes into effect gradually, as queued tasks execute naturally or are canceled by the user, rather than automatically canceling the excess queued tasks.
+
```

Comment:
The change from "queued for execution at any given point" to "queued for execution" would be erroneous, because it implies that you can only submit a fixed number of tasks over the lifetime of the executor.  The words "at any given point" are intended to convey the meaning that it is a maximum size of the queue.  I will agree to change it to "can be in the queue waiting for execution".
To go along with my reply to the earlier comment about unbounded/Integer.MAX_VALUE, I will make the same update here and remove 'unbounded'.
Other than the above, I am making the updates as suggested.